tests/integration/llm/prepare.py

```diff
@@ -625,6 +625,24 @@
         "option.tensor_parallel_degree": 4,
         "option.enable_chunked_prefill": "true",
     },
+    "llama-68m-speculative-medusa": {
+        "option.model_id": "JackFram/llama-68m",
+        "option.task": "text-generation",
+        "option.speculative_model": "abhigoyal/vllm-medusa-llama-68m-random",
```

Comment:
Non blocking: we should move these to s3 to avoid long term availability flakiness like we have seen with adapter models.

Reply:
Thanks Tyler. Uploaded llama-68m.